cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
I think you can delete this fixtures example for now. 
Also, fixtures are commonly used for mocking API intercepted responses, not sure if we are going to need that here.